google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.java

```diff
@@ -215,7 +211,7 @@ private boolean isValidConfigurationFile(String path) {
   /**
    * Validates the configuration and turns on/off any necessary warnings.
    */
-  private void validateConfiguration() {
+  private void toggleWarnings() {
```

Comment:
displayWarnings or showWarnings?  

'toggle' means to switch state, which I guess we don't always do here.